xoinvader/tests/test_animation.py

```diff
@@ -90,3 +92,57 @@ def test_animation_manager():
     time.sleep(1)
 
     assert animgr.update() is None
+
+
+@pytest.mark.parametrize(("values", "types", "expected"), (
+    ([1, 2, 3], int, True),
+    ([1, 2, "3"], (int, float), False),
+    ([1, 2, 3.0], int, False),
+    ([1, 2, 3.0], (int, float), True),
+))
+def test_same_type(values, types, expected):
+    assert animation.same_type(values, types) is expected
```

Comment:
Why not just add import of `same_type` to `from xoinvader.animation import ...`?